mesheryctl/internal/cli/root/perf/perf.go

```diff
@@ -51,31 +53,6 @@ var (
 	tokenPath          = ""
 )
 
-var perfDetails = `
```

Comment:
What will be the behaviour now if I run the command without any argument? Because @leecalcote asked to output the command description if user runs any sub-command without passing arguments.

Reply:
Displaying command definition after the error may seem bad UX, so either we should display a more informative message like
` No arguments passed, please see --help flag for details on the command`  or we should display the command details. Let's leave the final decision to @leecalcote .

Reply:
it's not customizable through cobra but let's see maybe we can create an error reporter for it.